Algorithms/Bit-Manipulation/UTF-8 encoding checker/UTF_8_encdng_chck.py

```diff
@@ -0,0 +1,10 @@
+s=input("Enter the string to check the utf-8 validation for: ")
+valid_utf8 = True
+try:
+    s.encode().decode('utf-8')
```

Comment:
Don't Use Direct python function.

Reply:
Please check. I have updated the code, removing any direct usage of inbuilt functions!